pages/submission/id.tsx

```diff
@@ -0,0 +1,49 @@
+import { NoPrescriptionIcon } from '../../components/icons';
+import { HomeLayout } from '../../components/layout/HomeLayout';
+import { SubmissionTextBox } from '../../components/submissions';
+import { Header } from '../../components/ui';
+const id = () => {
```

Comment:
I assume the component name is kind of a placeholder. In any case, this looks like you're gonna be using https://nextjs.org/docs/routing/dynamic-routes right?